web/js/codeworld_shared.js

```diff
@@ -1163,6 +1163,10 @@ function printMessage(type, message) {
     if (message.trim() === '') {
         return;
     }
+    // Deferred errors already handled on server side
```

Comment:
I think you should only handle this case if type === 'error', and you should match a more specific string like "(deferred type error)".  Otherwise, you're likely to ignore a lot of otherwise reasonable messages.

Reply:
Done.